apps/settings/style/icc.css

```diff
@@ -0,0 +1,28 @@
+#icc-entries[data-custom-icon] li > a {
+  padding-left: 5.5rem;
+}
+
+#icc-stk-list[data-custom-icon] li > a {
+  padding-left: 7rem;
+}
+
+#icc-entries[data-custom-icon] li > span,
+#icc-stk-list[data-custom-icon] li > span {
+  display: flex;
+  position: absolute;
+  width: 3rem;
+  height: 3rem;
+  left: 1.5rem;
+  top: calc(50% - 1.5rem);
+  box-sizing: content-box;
+  padding: 0;
+}
+
+#icc-stk-list[data-custom-icon] li > span {
+  left: 3rem;
+}
+
+#icc-entries[data-custom-icon] li > span > canvas,
+#icc-stk-list[data-custom-icon] li > span > canvas {
+  margin: auto;
+}
```

Comment:
Because I can't test this feature by our simcards, I use webIDE to hack the HTML to make sure the related CSS are applied. I think the final result should be something like this : 

![demo](https://cloud.githubusercontent.com/assets/68322/6053987/262d56fa-ad22-11e4-97bd-24d848d2b235.png)